sharding-orchestration/sharding-orchestration-center/sharding-orchestration-center-nacos/src/test/java/org/apache/shardingsphere/orchestration/center/instance/NacosConfigInstanceTest.java

```diff
@@ -80,7 +83,7 @@ public void assertPersist() {
     public void assertGet() {
         String value = "value";
         when(configService.getConfig(eq("sharding.test"), eq(group), anyLong())).thenReturn(value);
-        Assert.assertEquals(value, nacosConfigCenterRepository.get("/sharding/test"));
+        assertThat(value, is(nacosConfigCenterRepository.get("/sharding/test")));
```

Comment:
The first arg of assertThat is `actual`, second arg is `expected`.
It is reversed.